### PR TITLE
Add monitoring

### PR DESCRIPTION
**Pull request by peterthomassen**
This adds a Prometheus container which can be accessed through `www` (with basic auth). To keep the PR small, monitoring currently only includes Prometheus itself, and generic metrics from Django.

Next step is metrics for nginx and OpenVPN, but I guess this PR can be reviewed/merged independently while I work on these.

### CODE REVIEW

**Comment by peterthomassen** [comment]
I added basic nginx monitoring.

File: docker-compose.yml [hunk 1]
```diff
@@ -281,10 +284,39 @@ services:
         tag: "desec/openvpn-server"
     restart: unless-stopped
 
+  prometheus:
+    image: prom/prometheus:latest
+    init: true
+    volumes:
+    - ./prometheus/conf:/etc/prometheus:ro
+    - prometheus:/prometheus
```
Comment (nils-wisiol) [inline docker-compose.yml]: is there a difference in `./prometheus/` (two lines above) and `prometheus/` (line above)?

**Comment by nils-wisiol** [comment]
I confirmed it working on my machine

File: docker-compose.yml (hunk 1 above)
Comment (peterthomassen) [reply docker-compose.yml]: Yes! `prometheus` is the volume defined further down in the file, and `./prometheus` is a directory relative to the current directory. The volume carries persistent data, the other one is for mounting configuration.

**Comment by peterthomassen** [comment]
> Is there any data that needs to be deleted eventually? Otherwise, this lgtm

The [default retention time is 15 days](https://prometheus.io/docs/prometheus/latest/storage/#operational-aspects). We can adjust it later as we see fit.

**Pull request merged**